logbook/handlers.py

```diff
@@ -15,6 +15,9 @@
 import stat
 import errno
 import socket
+import gzip
+import math
+from brotli import Compressor
```

Comment:
This should be an optional dependency, and trying to instantiate a Brotli handler if it is missing should raise a RuntimeError